x-pack/plugins/xpack_main/public/services/path.js

```diff
@@ -11,6 +11,9 @@ export function PathProvider($window) {
   return {
     isLoginOrLogout() {
       return path === '/login' || path === '/logout';
+    },
+    isStatusPage() {
```

Comment:
@kobelb if we wait to merge this until after #23890 merges, then I can add to the new `unauthenticated` function, which will clean this up, and simplify a couple other checks in this PR

Reply:
That sounds great.